### PR TITLE
Ukrainian localization

### LINKED ISSUE #45: Translate messages in PO file into Ukrainian

### PR DESCRIPTION
**Pull request by mriabchenko**
Ukrainian localization feature

django.po localization file has been edited. English phrases has been translated into ukrainian.

This PR has been made to provide ukrainian localization for crm's user interface.

Resolves: #45

### CODE REVIEW

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -3,138 +3,139 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: \n"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Specify 1.0 here plz

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3,138 +3,139 @@
...
 "POT-Creation-Date: 2016-07-06 13:54+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"PO-Revision-Date: 2016-07-07 00:36+0400\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language-Team: \n"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Fill this in

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -3,138 +3,139 @@
...
+"X-Generator: Poedit 1.8.8\n"
+"Last-Translator: \n"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Add yourself

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -3,138 +3,139 @@
...
 
 #: SMSFlyCRM/SMSApp/forms/task.py:93
 msgid "Poll place"
-msgstr ""
+msgstr "Виборча дільниця"
 
 #: SMSFlyCRM/SMSApp/forms/task.py:96 SMSFlyCRM/SMSApp/forms/task.py:153
 msgid "Contact"
-msgstr ""
+msgstr "Контакти"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po:98]: This field actually specifies the last time guys touched/contacted specific person and actions taken then.

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -3,138 +3,139 @@
...
 
 #: SMSFlyCRM/SMSApp/forms/task.py:151
 msgid "Project"
-msgstr ""
+msgstr "Проект"
 
 #: SMSFlyCRM/SMSApp/forms/task.py:204
 msgid "End date"
-msgstr ""
+msgstr "Дата закінчення"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: завершення

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -3,138 +3,139 @@
...
 
 #: SMSFlyCRM/SMSApp/models/alphaname.py:24
 msgid "{} ({}). Registered on {} by {}"
-msgstr ""
+msgstr "{} ({}). Зареєстрований на {} з {}"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po:140]: Bear in mind that the third param is date and the fourth is user id

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -145,257 +146,257 @@ msgstr ""
 #: SMSFlyCRM/SMSApp/templates/campaign/list-archived.html:10
 #: SMSFlyCRM/SMSApp/templates/sent-messages.html:37
 msgid "Noname"
-msgstr ""
+msgstr "Без ім'я"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: That is a placeholder for untitled (unspecified/missing) template

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/models/message.py:53
 msgid "{} to {}  ({})"
-msgstr ""
+msgstr "{} до {}  ({})"
 
 #: SMSFlyCRM/SMSApp/models/task.py:23
 msgid "On elector's birthday"
-msgstr ""
+msgstr "В день народження виборця"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Alternate vowel and consonant letters at the beginning of this sentence

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/alphaname/list.html:8
 msgid "New alphaname"
-msgstr ""
+msgstr "Нове альфа-ім'я"
 
 #: SMSFlyCRM/SMSApp/templates/alphaname/list.html:17
 msgid "Registration date"
-msgstr ""
+msgstr "Дата регістрації"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Є

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 msgid "Archive"
-msgstr ""
+msgstr "Архівувати"
 
 #: SMSFlyCRM/SMSApp/templates/campaign/list-active.html:27
 #: SMSFlyCRM/SMSApp/templates/campaign/list-active.html:42
 #: SMSFlyCRM/SMSApp/templates/campaign/list-archived.html:11
 msgid "N/A"
-msgstr ""
+msgstr "Немає даних"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Н/Д

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 msgid "never"
-msgstr ""
+msgstr "ніколи"
 
 #: SMSFlyCRM/SMSApp/templates/campaign/list-active.html:30
 #: SMSFlyCRM/SMSApp/templates/campaign/list-active.html:45
 #: SMSFlyCRM/SMSApp/templates/campaign/list-archived.html:14
 msgid "forever"
-msgstr ""
+msgstr "завжди"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: "завжди" то always
тут "вічно" / "довіку" / "без кінця"

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/campaign/list-active.html:38
 msgid "paused"
-msgstr ""
+msgstr "тимчасово зупинений"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: This won't fit same width on the web UI

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/campaign/list-base.html:9
 msgid "New template (event-driven)"
-msgstr ""
+msgstr "Новий шаблон (подійний)"
 
 #: SMSFlyCRM/SMSApp/templates/components/filters.html:14
 msgid "Registration address"
-msgstr ""
+msgstr "Адреса реєстрації"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: прописки

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/components/filters.html:52
 msgid "Campaign cost:"
-msgstr ""
+msgstr "Вартість кампанії:"
 
 #: SMSFlyCRM/SMSApp/templates/components/filters.html:55
 msgid "SMS price:"
-msgstr ""
+msgstr "Вартість SMS:"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: ціна

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/components/filters.html:67
 msgid "Search by phone number"
-msgstr ""
+msgstr "Пошук за номером телефону"
 
 #: SMSFlyCRM/SMSApp/templates/components/filters.html:70
 msgid "Preview recipients"
-msgstr ""
+msgstr "Перегляд одержувача"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: It's plural

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 16]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 #: SMSFlyCRM/SMSApp/templates/main.html:8
 msgid "Alphanames list"
-msgstr ""
+msgstr "Список альфа-імен"
 
 #: SMSFlyCRM/SMSApp/templates/components/navbar.html:5
 #: SMSFlyCRM/SMSApp/templates/main.html:9
 msgid "Campaign templates"
-msgstr ""
+msgstr "Шаблони кампанії"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: It's plural

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 17]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/components/text-buttons.html:7
 msgid "Address"
-msgstr ""
+msgstr "Адреса"
 
 #: SMSFlyCRM/SMSApp/templates/components/under-construction.html:4
 msgid "Achtung! This chapter is currently under construction!"
-msgstr ""
+msgstr "Увага! Ця сторінка знаходиться на стадії розробки"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Missing bang in the end

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 18]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/sent-messages.html:16
 msgid "Filters"
-msgstr ""
+msgstr "Фільтри"
 
 #: SMSFlyCRM/SMSApp/templates/sent-messages.html:17
 msgid "Sent at"
-msgstr ""
+msgstr "Надіслано в"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: о

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 19]
```diff
@@ -145,257 +146,257 @@ msgstr ""
...
 
 #: SMSFlyCRM/SMSApp/templates/sent-messages.html:18
 msgid "Receiver"
-msgstr ""
+msgstr "Отримувач"
 
 #: SMSFlyCRM/SMSApp/templates/sent-messages.html:21
 msgid "Cost"
-msgstr ""
+msgstr "Ціна"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: Вартість

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po (hunk 8 above)
Comment (mriabchenko) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: @webknjaz do you mean that it has to be like "В День Народження виборця"?

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po (hunk 16 above)
Comment (mriabchenko) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: @webknjaz i suppose that everything is ok here

Comment (mriabchenko) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po]: @webknjaz i will change it to "Шаблони кампаній"

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 20]
```diff
@@ -3,138 +3,141 @@
...
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"PO-Revision-Date: 2016-07-07 11:25+0400\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language-Team: uk\n"
+"X-Generator: Poedit 1.8.8\n"
+"Last-Translator: Nikolay_R\n"
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po:17]: Correct format is: `FULL NAME <EMAIL@ADDRESS>`

File: SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po [hunk 21]
```diff
@@ -3,138 +3,141 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (webknjaz) [inline SMSFlyCRM/SMSApp/locale/uk_UA/LC_MESSAGES/django.po:4]: ?

**Comment by webknjaz** [comment]
@nikolayr1  ping?

**Pull request closed without merging**